src/quo/components/overlay/style.cljs

```diff
@@ -0,0 +1,17 @@
+(ns quo.components.overlay.style)
+
+(defn overlay-background
+  [background-color]
+  {:position         :absolute
+   :top              0
+   :left             0
+   :right            0
+   :bottom           0
+   :background-color background-color})
+
+(def container
+  {:flex 1})
+
+(def blur-container
+  {:flex             1
+   :background-color :transparent})
```

Comment:
@smohamedjavid I have to pass this style because of this issue in blur library https://github.com/Kureev/react-native-blur/issues/483#issuecomment-1328034059

Reply:
Noted. Thanks for the explanation!